<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>批量PING</string>
  </property>
  <property name="windowIcon">
   <iconset resource="assets/assets.qrc">
    <normaloff>:/icons/favicon.png</normaloff>:/icons/favicon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QWidget" name="titleWidget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1,1">
       <item>
        <widget class="QWidget" name="widget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="backBtn">
            <property name="minimumSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: rgba(0, 0, 0, 0);
	border-radius: 5px 5px 5px 5px;
    border: none;
}

QPushButton:hover {
    background-color: rgba(0, 0, 0, 0.1);
	border-radius: 5px 5px 5px 5px;
    border: none;
}

QPushButton:pressed {
    background-color: rgba(0, 0, 0, 0.3);
	border-radius: 5px 5px 5px 5px;
    border: none;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_4">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>246</width>
              <height>45</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="titleLabel">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>48</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>48</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
         <property name="text">
          <string>批 量 PING</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_2" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>246</width>
              <height>45</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="settingsBtn">
            <property name="minimumSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
            <property name="toolTip">
             <string>设置</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: rgba(0, 0, 0, 0);
	border-radius: 5px 5px 5px 5px;
    border: none;
}

QPushButton:hover {
    background-color: rgba(0, 0, 0, 0.1);
	border-radius: 5px 5px 5px 5px;
    border: none;
}

QPushButton:pressed {
    background-color: rgba(0, 0, 0, 0.3);
	border-radius: 5px 5px 5px 5px;
    border: none;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="exportBtn">
            <property name="minimumSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>36</width>
              <height>36</height>
             </size>
            </property>
            <property name="toolTip">
             <string>数据导出（.xlsx）</string>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: rgba(0, 0, 0, 0);
	border-radius: 5px 5px 5px 5px;
    border: none;
}

QPushButton:hover {
    background-color: rgba(0, 0, 0, 0.1);
	border-radius: 5px 5px 5px 5px;
    border: none;
}

QPushButton:pressed {
    background-color: rgba(0, 0, 0, 0.3);
	border-radius: 5px 5px 5px 5px;
    border: none;
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="iconSize">
             <size>
              <width>32</width>
              <height>32</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="whatsThis">
       <string/>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="assets/assets.qrc"/>
 </resources>
 <connections/>
</ui>
